script/setup/install-runc

```diff
@@ -21,13 +21,15 @@
 set -eu -o pipefail
 
 function install_runc() {
-	RUNC_COMMIT=$(grep opencontainers/runc "$GOPATH"/src/github.com/containerd/containerd/go.mod | awk '{print $2}')
+	# When updating RUNC_COMMIT, consider updating the runc module as well
+	: ${RUNC_COMMIT:=12644e614e25b05da6fd08a38ffa0cfe1903fdec} # v1.0.0-rc93
```

Comment:
what is a good way to publicize this? we want to highlight this well enough for folks who have now been used to picking up SHA from the go.mod

Reply:
chase it down with hound and push commits ?   :-)

Reply:
> chase it down with hound and push commits ? :-)

Hehehe!

Reply:
Should we change the name of the variable to `RUNC_VERSION` to encourage using tagged releases (and default to using a tag)?